### PR TITLE
Custom translation nmt

### PR DESCRIPTION
**Pull request by ydharavath**
Support for custom translation and "Do Not Translate" (DNT) phrases is added for nmt. 
Custom translations: Users can specify word or phrase pairs separated by double hash (##) in a line for custom translation handling.
DNT phrases: Words or phrases that should remain untranslated can be listed in the same file in separate lines.

### CODE REVIEW

File: WORKSPACE
Comment (ydharavath) [inline WORKSPACE:1]: Change made for testing and to be updated before merge

File: riva/clients/nmt/riva_nmt_t2t_client.cc
Comment (rmittal-github) [inline riva/clients/nmt/riva_nmt_t2t_client.cc]: can we name the function better? since it is dnt phrases file, not user dictionary

File: riva/clients/nmt/riva_nmt_t2t_client.cc
Comment (rmittal-github) [inline riva/clients/nmt/riva_nmt_t2t_client.cc]: this will get updated as per `dnt_phrases`

File: riva/clients/nmt/riva_nmt_t2t_client.cc
Comment (rmittal-github) [inline riva/clients/nmt/riva_nmt_t2t_client.cc]: better to use ## here to allow multiple spaces in the key/value

**Comment by rmittal-github** [comment]
let's also push python-clients MR

File: riva/clients/nmt/riva_nmt_t2t_client.cc
Comment (rmittal-github) [inline riva/clients/nmt/riva_nmt_t2t_client.cc]: suggest to handle key == empty case on server side, so that we filter out the case if user's custom clients still send empty key

**Comment by rmittal-github** [comment]
LGTM, will wait for CI testing to complete before merge

**Review by rmittal-github — APPROVED** [review]

**Pull request merged**
